metron-streaming/Metron-Common/src/test/java/org/apache/metron/enrichment/EnrichmentConfigTest.java

```diff
@@ -0,0 +1,253 @@
+package org.apache.metron.enrichment;
+
+import org.apache.metron.Constants;
+import org.apache.metron.domain.SourceConfig;
+import org.apache.metron.utils.JSONUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class EnrichmentConfigTest {
+  @Test
+  public void testThreatIntel() throws Exception {
+    /*
+    {
+      "index": "bro",
```

Comment:
Normally I'd agree, but those comments are there because multiline strings in java are lacking, so if changes to the JSON structure happen, the changes generally happen to the comment and then the string is replaced, letting the IDE handle escaping quotes.  One thing that we could consider is incorporating something like https://github.com/benelog/multiline, which i've seen in other projects for these kinds of situations.